packs/equipment/tacticians-helm.json

```diff
@@ -43,16 +43,26 @@
                 "selector": "warfare-lore",
                 "type": "item",
                 "value": 1
+            },
+            {
+                "key": "Note",
+                "outcome": ["success", "criticalSuccess"],
+                "predicate": [
+                    "encounter",
+                    {
+                        "not": "self:participant:own-turn"
+                    }
```

Comment:
```suggestion
                    "reactive-strike"
```

Reply:
I had intentionally left this out of the predicate because it's a deceptively complex thing to detect. There are at least six different ways that reactive strike/AoO can show up in the roll options right now:

- `feature:reactive-strike`, for a remaster fighter
- `feature:attack-of-opportunity`, for a legacy fighter
- `feat:attack-of-opportunity` for the Attack of Opportunity feats for classes like Champion, Barbarian, and Magus
- `feat:reactive-striker`, for a character using the remaster version of the fighter archetype
- `feat:opportunist`, for a character using the legacy fighter archetype
- Not at all, for niche edge cases like the Viper's Fang talisman

...and that's just for player characters

Reply:
Oh, I was thinking of retributive strike which I added a toggle for the other day, name change is going to mess with me for a while. This is still wrong for the other ways to get off turn strikes like retributive strike and opportune backstab. But I guess since the text specifies and it would only show up if someone with multiple reaction strikes is wearing the helmet which is going to be niche enough that we don't need to care.

Reply:
Though, it isn't impossible to get a reactive strike during your own turn if someone's reaction or free action involves moving.

Reply:
That's also a good point. I think reactive strikes on your own turn are uncommon enough that it's not worth making it a general note on every strike and cluttering the UI.